src/Illuminate/Database/Eloquent/Relations/HasOne.php

```diff
@@ -2,18 +2,38 @@
 
 namespace Illuminate\Database\Eloquent\Relations;
 
+use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Collection;
 
 class HasOne extends HasOneOrMany
 {
     /**
+     * Determine whether getResults should return a default new model instance or not.
```

Comment:
This variable doesn't determine anything, rather, the description should be just "If getResults should return a default model instance."